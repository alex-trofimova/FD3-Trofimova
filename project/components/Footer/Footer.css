.Footer {
  height: 65px;
  background-color: rgb(45, 79, 156);
  color: white;
}

.Footer_wrapper {
  position: relative;
  height: 100%;
  margin: 0 auto;
}
