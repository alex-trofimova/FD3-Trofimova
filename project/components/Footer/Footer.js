import React from 'react';
//import PropTypes from 'prop-types';

import './Footer.css';

class Footer extends React.PureComponent {

  static propTypes = {
    //name: PropTypes.string.isRequired,
  };

  render() {

    return (
        <footer className="Footer">
          <div className="Footer_wrapper">
            @Trofimova 2021
          </div>
          
        </footer>      
    )
    ;

  }

}

export default Footer;
