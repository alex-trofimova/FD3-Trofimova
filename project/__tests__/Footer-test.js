import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../components/Footer/Footer';

test('работа Footer', () => {

  const component = renderer.create(
    <Footer />
  );

  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
    
});
