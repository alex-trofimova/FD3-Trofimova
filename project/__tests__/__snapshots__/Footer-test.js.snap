// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа Footer 1`] = `
<footer
  className="Footer"
>
  <div
    className="Footer_wrapper"
  >
    @Trofimova 2021
  </div>
</footer>
`;
